README.md

```diff
@@ -69,6 +71,9 @@ A template of the `licenses.json` file can be found in the repository at `src/ma
    # according to the template at 'src/main/resources/licenses.json')
    # > option "scanner": take conflicts from the scanner response
     license_conflict=json
+   
+   # Alternatively, you can supply all the necessary values associated with github and license using these env variables:
+   # LPVS_GITHUB_LOGIN, LPVS_GITHUB_TOKEN, LPVS_GITHUB_API_URL, LPVS_GITHUB_SECRET, LPVS_LICENSE_FILEPATH and LPVS_LICENSE_CONFLICT.
```

Comment:
Also please add env variables names inro application.properties as a comment (#) before the names of variables in the file (users should understand the correspondence between env variable name and LPVS variable name).
Example:

# GitHub settings

# Corresponding env. variable is LPVS_GITHUB_LOGIN
github.login=